docs/getting_started/components/sip.md

```diff
@@ -0,0 +1,32 @@
+# SIP
+
+Makes a SIP connection with specified provider.
+From this connection it consumes a remote RTP stream with G711 A-law audio
+and publishes through this connection all audio tracks from engine.
```

Comment:
Engine isn't a Jellyfish term
```suggestion
and publishes through this connection all audio tracks from the [Room](REFLINK_TO_ROOM).
```

Reply:
One more thing: the last PR added a new `Room` page. Would you be so kind and change all occurrences in File, HLS and RTSP to point to that?